content/specs/data-model.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Data Model
+summary: Consistent data encoding for records and messages.
+---
+
+# Data Model
+
+Records and messages in atproto are stored, transmitted, encoded, and authenticated in a consistent way. The core "data model" is based on [Interplanetary Linked Data (IPLD)](https://ipld.io/docs/data-model/), a specification for hash-linked data structures from the IPFS ecosystem.
+
+When data needs to be authenticated (signed), referenced (linked by content hash), or stored efficiently, it is encoded in Concise Binary Object Representation (CBOR). CBOR is an IETF standard roughly based on JSON. IPLD specifies a normalized subset of CBOR called "DAG-CBOR", which is what atproto uses.
+
+IPLD also specifies an analogous set of conventions of JSON called "DAG-JSON", but atproto uses a different set of conventions when encoding JSON data.
```

Comment:
Thanks for this! I went off and implemented https://github.com/snarfed/dag-json a while back due to misreading the `car inspect` output and thinking the PDS itself was returning DAG-JSON. Funny, and totally PEBKAC.